babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -1776,6 +1776,56 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test void testIndexWithoutName() {
+    final String sql = "create table foo (bar integer) index (bar)";
+    final String expected = "CREATE TABLE `FOO` (`BAR` INTEGER) INDEX (`BAR`)";
+    sql(sql).ok(expected);
+  }
+
+  @Test void testIndexWithName() {
+    final String sql = "create table foo (bar integer) index baz (bar)";
+    final String expected = "CREATE TABLE `FOO` (`BAR` INTEGER) "
+        + "INDEX `BAZ` (`BAR`)";
+    sql(sql).ok(expected);
+  }
+
+  @Test void testUniqueIndexWithoutName() {
+    final String sql = "create table foo (bar integer) unique index (bar)";
+    final String expected = "CREATE TABLE `FOO` (`BAR` INTEGER) "
+        + "UNIQUE INDEX (`BAR`)";
+    sql(sql).ok(expected);
+  }
+
+  @Test void testUniqueIndexWithName() {
+    final String sql = "create table foo (bar integer) unique index baz (bar)";
+    final String expected = "CREATE TABLE `FOO` (`BAR` INTEGER) "
+        + "UNIQUE INDEX `BAZ` (`BAR`)";
+    sql(sql).ok(expected);
+  }
+
+  @Test void testMulticolumnIndex() {
+    final String sql = "create table foo (bar integer, qux integer) "
+        + "index (bar, qux)";
+    final String expected = "CREATE TABLE `FOO` (`BAR` INTEGER, `QUX` INTEGER) "
+        + "INDEX (`BAR`, `QUX`)";
+    sql(sql).ok(expected);
+  }
+
+  @Test void testMultipleIndices() {
+    final String sql = "create table foo (bar integer, qux integer) "
+        + "index (bar), unique index (qux)";
+    final String expected = "CREATE TABLE `FOO` (`BAR` INTEGER, `QUX` INTEGER) "
+        + "INDEX (`BAR`), UNIQUE INDEX (`QUX`)";
+    sql(sql).ok(expected);
+  }
+
+  @Test void testPrimaryIndexWithSecondaryIndex() {
+    final String sql = "create table foo (bar integer, qux integer) "
+          + "primary index (bar), index (qux)";
+    final String expected = "CREATE TABLE `FOO` (`BAR` INTEGER, `QUX` INTEGER) "
+          + "PRIMARY INDEX (`BAR`), INDEX (`QUX`)";
+    sql(sql).ok(expected);
+
```

Comment:
Missing close bracket.

Reply:
Yeah, I must have accidentally deleted it when I was trying to fix conflicts.